providers/flagd-in-process/pkg/provider.go

```diff
@@ -127,7 +127,7 @@ func NewProvider(ctx context.Context, opts ...ProviderOption) *Provider {
 		provider.logger,
 		s,
 		eval.WithEvaluator(
-			"fractionalEvaluation",
+			"fractional",
```

Comment:
This is the only functional change.

Reply:
Will this be a breaking change 🤔  ?

Reply:
sort-of... but it was a bug in the first place. As per spec/doc the operator was always supposed to be called `fractional`.

Also we haven't released it yet :smile:

Reply:
Ahh yeah, sorry I forgot the fact that the Go in-process provider is a separate package :)